pkg/chartutil/create.go

```diff
@@ -149,16 +173,13 @@ const defaultIgnore = `# Patterns to ignore when building packages.
 .project
 .idea/
 *.tmproj
+.vscode/
 `
 
 const defaultIngress = `{{- if .Values.ingress.enabled -}}
 {{- $fullName := include "<CHARTNAME>.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
 apiVersion: networking.k8s.io/v1
-{{- else -}}
-apiVersion: extensions/v1beta1
```

Comment:
I'm curious how come this was removed?  Won't v3 need to work with clusters earlier than 1.14?

Reply:
I would say the change is in line with the official Kubernetes version support policy (https://kubernetes.io/docs/setup/release/version-skew-policy/). Personally, I never liked having conditional logic for API versions in the starter chart. After all, this is not about Helm supporting older Kubernetes versions. It's about creating a starter chart which you will have to adjust to your needs anyways.

Reply:
Thanks for the clarification.  I guess I was surprised that this conditional API version logic was added 2 weeks ago and now removed.  But I had missed that this was the charter start code when I skimmed the change.